.devcontainer/install-dependencies.sh

```diff
@@ -210,6 +210,15 @@ if should-install "$TOOL_DEST/cmctl"; then
     curl -L "https://github.com/jetstack/cert-manager/releases/latest/download/cmctl-${os}-${arch}.tar.gz" | tar -xz -C "$TOOL_DEST"
 fi
 
+BUILDX_DEST=$HOME/.docker/cli-plugins
+write-verbose "Checking for $BUILDX_DEST/docker-buildx"
+if should-install "$BUILDX_DEST/docker-buildx"; then
+    write-info "Installing buildx-${os}_${arch}…"
```

Comment:
The variables `os` and `arch` aren't defined when I check using **bash** under Ubuntu - so unless there's some magic going on, it looks like these will only work when building the devcontainer - but this script is supposed to work for local setup as well.

Reply:
Aren't these vars (os and arch) are set above on line 205?